src/components/ProjectDetailPage/ProjectDetailPage.tsx

```diff
@@ -16,6 +17,8 @@ export const ProjectDetailPage = () => {
     projectService.getProject({ id: projectId as string }, requestConfig)
   );
 
+  useTitle(dataContainer.data?.name + ' | Project');
```

Comment:
1) We prefer using [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in cases like this

```
useTitle(`${dataContainer.data?.name} | Project`)
```